IronFoundry.Container.Test/Utilities/FileSystemEffectiveAccessComputerTests.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.IO;
+using System.Security.AccessControl;
+using System.Security.Principal;
+using Xunit;
+
+namespace IronFoundry.Container.Utilities
+{
+    public class FileSystemEffectiveAccessComputerTests : IDisposable
+    {
+        TempFile TestFile { get; set; }
+        FileSystemEffectiveAccessComputer EffectiveAccess { get; set; }
+        IdentityReference CurrentIdentity { get; set; }
+
+        public FileSystemEffectiveAccessComputerTests()
+        {
+            TestFile = new TempFile(Path.GetTempPath());
+            EffectiveAccess = new FileSystemEffectiveAccessComputer();
+            CurrentIdentity = WindowsIdentity.GetCurrent().User;
+        }
+
+        public void Dispose()
+        {
+            TestFile.Dispose();
+        }
+
+        public class ComputeAccess : FileSystemEffectiveAccessComputerTests
+        {
+            [Fact(Skip = "Fails on domain joined machines")]
```

Comment:
(I added the Skip when I rebased)
These tests all fail on my machine because I'm logged in as my domain account, but don't have access to the domain controller.

There should be a way to compute effective access on files without needing to contact the DC.  Let's investigate this when I'm in the office.  Obviously software is able to perform access checks on my machine when I'm not connected, so maybe we are missing something.